Makefile

```diff
@@ -103,6 +102,6 @@ set_env_class_prod:
 vagrant-deploy:
 	vagrant/deploy.sh
 
-.PHONY: deploy_pipelines
-deploy_pipelines:
+.PHONY: pipelines
+pipelines: ## Upload pipelines to Concourse
```

Comment:
Check now added in https://github.com/alphagov/paas-cf/pull/165/commits/4dbb4e01b361f1582b37d3b3a436f8d0caf1797e